base/sparse/sparsevector.jl

```diff
@@ -1094,16 +1083,6 @@ macro unarymap_z2nz(op, TF)
     end)
 end
 
-for op in [:exp, :exp2, :exp10, :log, :log2, :log10,
-           :cos, :csc, :cot, :sec, :cospi,
-           :cosd, :cscd, :cotd, :secd,
-           :acos, :acot, :acosd, :acotd,
-           :cosh, :csch, :coth, :sech,
-           :acsch, :asech]
-    @eval @unarymap_z2nz $(op) Number
```

Comment:
Absolutely, chances are most of the `Xmap_YZ` macros/functions for `SparseVector`s can be cleaned out with some care :). Would a separate pull request be alright?